app/views/incomeTaxComparison/IncomeSummary.scala.html

```diff
@@ -64,22 +64,22 @@ <h2 id="incomeSummaryHeading" class="heading-large">@{incomeSummaryHeading}</h2>
         @for((employmentIncomeSourceDetail, count) <- incomeSourceViewModel.employmentIncomeSourceDetail.zipWithIndex) {
 
             <tr>
-                <td scope="row" role="gridcell" class="table-title">
+                <td scope="row" role="table" class="table-title">
```

Comment:
please make this and others role="cell"
like [this](https://github.com/Fenwick17/nhs-responsive-tables/commit/7b12f3c965922acb443c3f31dd08eb2a0d706a7a)